src/net.cpp

```diff
@@ -2529,7 +2530,7 @@ std::vector<CAddress> CConnman::GetAddresses(Optional<Network> requestor_network
 {
     if (requestor_network) {
         const std::chrono::microseconds current_time = GetTime<std::chrono::microseconds>();
-        const Network network = requestor_network.get();
+        const Network network = requestor_network.get_value_or(NET_UNROUTABLE);
```

Comment:
This seems unnecessary. `get_value_or` only matters if the Optional is nullopt and a nullopt can't reach this code path.